content/meetings/Meeting-2019-09-26.rst

```diff
@@ -20,54 +20,59 @@ RSVP via meetup.com here:
 
 Location
 --------
-TBD
+
+LinkedIn Corporation, Unify Meeting Room
+950 West Maude Ave
+Sunnyale, CA
 
 
 Meeting Details
 ---------------
 Meeting Schedule:
 
 * 7:00 pm Food and Announcements
-* 7:10 pm Talks start
+* 7:15 pm Talks start
 * 8:30 pm Networking
 * 9:00 pm Event ends
 
-Pigs, Pythons and Goats: The Animal Guide to TDD
-------------------------------------------------
-
-There are often many buzzwords that are used incorrectly and abused. Often
-people hear about Test Driven Development (TDD) and think this is just a term
-that means "Testing your code." Although Testing your code is important, Test
-Driven Development is a special paradigm that is often used in certain
-companies.
-
-After this talk, you may decide that full Test Driven Development (TDD) isn’t
-for you (or, it may be a turning point for you). Regardless, you should fully
-understand the paradigms and what it is - as it will come up in discussions
-some time in your career.
+Combining compiling technology with AI / Also GitPython and cProfile Packages
+-----------------------------------------------------------------------------
 
-To make it accessible to everyone, we will start this talk with an introduction
-to Unit Tests and testing frameworks (e.g, unittest, pytest, nose, etc.) We do
-a few very basic live demonstrations of both succeeding and failing unit tests.
-
-We quickly, however ramp up building a data structure (e.g., stack, queue,
-tree, etc.) using a full Red, Green, Refactor TDD cycle (cycles within Test
-Driven Development). This will be somewhat interactive and there will be a lot
-of slides with Python code.
+Everything in software development is a type of automation, from timecards
+stamps to sorting clients’ last name. Compilers are the most powerful softwares
```

Comment:
The joint mind of stackexchange says that `software` is uncountable.
https://english.stackexchange.com/questions/4144/what-is-the-plural-form-of-software

Reply:
@o0-o0 Ah, thank you. This is the Abstract given by the user. We typically don't correct the Abstract that the user sends.

More often, the things that we look for is, "Did Glen get the date wrong?"  "Did Glen link to the wrong event," etc.

It is interesting though. Going forward, it may not be a bad idea to give this feedback to the user for their abstracts. 

I'm going to go ahead and make this change today.

Reply:
I see.
Apart from that everything looks good to me.

Reply:
Updated.